src/modules/categories/styles/list.scss

```
@@ -0,0 +1,49 @@
+@use '../../../styles/utils/media' as *;
+@use '../../../styles/utils/variables' as *;
+@use '../../../styles/utils/typography' as *;
```

тяни всегда вот так в таких блоках
```suggestion
@use '@/styles/utils/index' as *;
```